src/rct2/S6Exporter.cpp

```diff
@@ -562,6 +562,7 @@ extern "C"
         SDL_WriteU8(rw, gCheatsDisableLittering);
         SDL_WriteU8(rw, gCheatsNeverendingMarketing);
         SDL_WriteU8(rw, gCheatsFreezeClimate);
+        SDL_WriteU8(rw, gCheatsDisablePlantAging);
```

Comment:
Changes to this part require bumping [network version](https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/network/network.h#L58). There's another PR inbound also touching it, https://github.com/OpenRCT2/OpenRCT2/pull/3782, they can either go in together or there'll be minor conflict requiring rebase.

@IntelOrca any comments?